eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+        if Path(input_file).suffix == '.newick': 
+            self.file_path = input_file
+            self.file_path_label.destroy()  # overwrites the old file path in the grid system
+            self.file_path_label = tk.Label(recon_input_info_frame, text=input_file)
+            self.file_path_label.grid(row=0, column=0, sticky="w")
+            # this is only DEMO for now
+            host_tree_info = tk.Label(recon_input_info_frame, text="Host:  83 tips (DEMO)")
+            host_tree_info.grid(row=1, column=0, sticky="w")
+            parasite_info = tk.Label(recon_input_info_frame, text="Parasite/symbiont:  78 tips (DEMO)")
+            parasite_info.grid(row=2, column=0, sticky="w")           
+        else:
+            messagebox.showinfo("Warning", "Please load a '.newick' file.")
+
+        # enables the next step, setting DTL costs
+        if self.file_path != "": 
+            self.view_cost_btn.configure(state=tk.NORMAL)
+            self.DTL_cost()
+
+    def DTL_cost(self):
+        """Sets DTL costs by clicking on the matplotlib graph or by entering manually."""
+        # creates a frame for setting DTL costs
+        costs_frame = tk.Frame(self.output_frame)
+        costs_frame.grid(row=1, column=0, sticky="nsew")
+        costs_frame.grid_columnconfigure(0, weight=1)
+        costs_frame.grid_columnconfigure(1, weight=3)
+        costs_frame.grid_rowconfigure(0, weight=1)
+        costs_frame.grid_rowconfigure(1, weight=1)
+        costs_frame.grid_rowconfigure(2, weight=1)
+        costs_frame.grid_propagate(False)
+
+        dup_label = tk.Label(costs_frame, text="Duplication:")
+        trans_label = tk.Label(costs_frame, text="Transfer:")
+        loss_label = tk.Label(costs_frame, text="Loss:")
+        dup_label.grid(row=0, column=0, sticky="w")
+        trans_label.grid(row=1, column=0, sticky="w")
+        loss_label.grid(row=2, column=0, sticky="w")
+        self.dup_cost = tk.StringVar()
+        self.trans_cost = tk.StringVar()
+        self.loss_cost = tk.StringVar()
+        dup_entry = tk.Entry(costs_frame, width=3, textvariable=self.dup_cost)
+        trans_entry = tk.Entry(costs_frame, width=3, textvariable=self.trans_cost)
+        loss_entry = tk.Entry(costs_frame, width=3, textvariable=self.loss_cost)
+        dup_entry.bind('<Return>', self.check_dup_input)
+        trans_entry.bind('<Return>', self.check_trans_input)
+        loss_entry.bind('<Return>', self.check_loss_input)
+        #dup_entry.bind('<FocusOut>', self.check_dup_input)  # (bug) this will make the invalid input warning appear twice
+        #trans_entry.bind('<FocusOut>', self.check_trans_input)  # same as above 
+        #loss_entry.bind('<FocusOut>', self.check_loss_input)  # same as above 
+        dup_entry.grid(row=0, column=1, sticky="w")
+        trans_entry.grid(row=1, column=1, sticky="w")
+        loss_entry.grid(row=2, column=1, sticky="w")
+
+    def plot_cost_landscape(self):
```

Comment:
I think this is called `cost_regions` in other parts of the code. Use this name convention over `cost_landscape`

Reply:
Also modify the comments.

Reply:
Done:)